### PR TITLE
Update CSS3 Readme

### PR DESCRIPTION
**Pull request by oxleberry**
https://github.com/gdisf/teaching-materials/issues/449

# Summary

Fixes issue #449 

Here's a description of changes:

* Updated CSS3 Readme

_Before you merge, make sure you visually check your changes in the deploy preview. A link to the preview will appear in the PR comments when ready._

cc @gdisf/admins

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview ready!

Built with commit ca6bfddd0555f5b7ec72bfb867f5611fb954f04c

https://deploy-preview-507--teaching-materials.netlify.com

File: css3-selectors/README.md [hunk 1]
```diff
@@ -1,29 +1,41 @@
...
+- CSS attribute selectors
+- CSS3 form pseudo-classes
+- CSS3 structural selectors
+- CSS3 pseudo-elements
 
-The workshop will be a mix of lectures and hands-on exercises, and will be taught by [teacher X, tiny bio]. This is a great follow-up workshop for those who have attended our HTML/CSS workshop.
+**Slides**
 
-### Prerequisites:
+https://www.teaching-materials.org/css3-selectors/
```
Comment (trucnguyen) [inline css3-selectors/README.md]: Nit: make this a markdown link

Comment (oxleberry) [inline css3-selectors/README.md]: ok, I'm on it.

**Comment by trucnguyen** [comment]
Closing this PR because you submitted another that fixed my comment :)

**Pull request closed without merging**